images/pytest/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /app
+
+COPY metrics/get_jq.sh /app/get_jq.sh
+COPY requirements3.txt /app/requirements3.txt
+
+RUN /bin/bash -c "apt-get update && apt-get install -y make wget"
+RUN /bin/bash -c "pip3 install -r requirements3.txt"
```

Comment:
When a PR updates `requirements3.txt`, do we expect `make pytests` also run `pip3 install -r requirements3.txt`?

Reply:
Any idea?